packages/babel-plugin-open-source/script.js

```diff
@@ -4,7 +4,7 @@ if (typeof document !== 'undefined') {
     if (!event.altKey) return;
 
     event.preventDefault();
-    const { filename, start, editor } = JSON.parse(event.target.dataset.source)
-    window.open(editor + '://file/' + filename + ':' + start)
+    const { filename, start, url } = JSON.parse(event.target.dataset.source)
```

Comment:
we can remove filename + start from the payload and plugin completely if it's unused now?